crates/shared/Cargo.toml

```diff
@@ -9,54 +9,51 @@ license = "MIT OR Apache-2.0"
 doctest = false
 
 [dependencies]
-anyhow = "1.0"
-assert_approx_eq = "1.1"
+anyhow = { workspace = true }
 async-stream = "0.3"
-async-trait = "0.1"
+async-trait = { workspace = true }
 atty = "0.2"
-cached = { version = "0.34", default-features = false }
+cached = { workspace = true }
 chrono = { version = "0.4", default-features = false, features = ["clock"] }
-clap = { version = "3.1", features = ["derive", "env"] }
+clap = { workspace = true }
 contracts = { path = "../contracts" }
 database = { path = "../database" }
-derivative = "2.2"
-ethcontract = { version = "0.21.0", default-features = false }
-ethcontract-mock = { version = "0.21.0", default-features = false }
-futures = "0.3"
-gas-estimation = { git = "https://github.com/cowprotocol/gas-estimation", tag = "v0.7.1", features = ["web3_", "tokio_"] }
+derivative = { workspace = true }
+ethcontract = { workspace = true }
+ethcontract-mock = { workspace = true }
+futures = { workspace = true }
+gas-estimation = { workspace = true }
 global-metrics = { path = "../global-metrics" }
-hex = { version = "0.4", default-features = false }
-hex-literal = "0.3"
-http = "0.2.6"
```

Comment:
This was just used for `HeaderMap`s with `reqwest`. However, `reqwest` already re-exports these types for easier use, so use the re-export instead.